pywal/__main__.py

```diff
@@ -175,6 +178,17 @@ def parse_args(parser):
     if args.R:
         colors_plain = theme.file(os.path.join(CACHE_DIR, "colors.json"))
 
+        if args.l:
+            wallpaper_cache = open(os.path.join(CACHE_DIR, "wal"))
```

Comment:
Swap this (and below) to use: `util.read_file`.

Reply:
Done!